README.md

```diff
@@ -60,16 +60,22 @@ E-Bikes is a sample application that demonstrates how to build applications with
     sfdx force:user:permset:assign -n ebikes
     ```
 
-1. Load sample data:
+1. Import sample data:
 
     ```
-    sfdx force:data:tree:import --plan ./data/sample-data-plan.json
+    sfdx force:data:tree:import -p ./data/sample-data-plan.json
```

Comment:
Same as above.

Reply:
Yes, we weren't consistent with commands flags in the readme. Most commands used the short flags so I decided to stick to it.